external/fv3fit/fv3fit/_shared/taper_function.py

```diff
@@ -13,6 +13,13 @@ def taper_mask(
     return xr.where(novelty_score > cutoff, 0, 1)
 
 
+def taper_nan_mask(
```

Comment:
Where is this used? I assume this is listed in a yaml someplace.

Reply:
Yes, there is a configuration option for what "taper" function to use.